prep-html-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
Oops! You are missing your `head` element. Even though we don't have any content that needs to go in it right now, it is still a mandatory part of any HTML document.